cmd/killTask.go

```diff
@@ -0,0 +1,74 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cmd
+
+import (
+	"strconv"
+
+	realis "github.com/aurora-scheduler/gorealis/v2"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(killTask)
+
+	/* Sub-Commands */
+
+	// Kill Job
+	killTask.AddCommand(killTaskCmd)
+
+	killTaskCmd.Flags().StringVarP(env, "environment", "e", "", "Aurora Environment")
+	killTaskCmd.Flags().StringVarP(role, "role", "r", "", "Aurora Role")
+	killTaskCmd.Flags().StringVarP(name, "name", "n", "", "Aurora Name")
+	killTaskCmd.Flags().StringVarP(instance, "instance", "i", "", "Instance number")
```

Comment:
instance number must be slice []int32 not string.
We can kill a list of instances in the same job.